subsys/bluetooth/audio/tbs_client.c

```diff
@@ -0,0 +1,2166 @@
+		tbs_client_cbs->remote_uri(conn, 0, tbs_inst->index, uri);
+	}
+}
+
+static void friendly_name_notify_handler(struct bt_conn *conn,
+					 const struct bt_tbs_instance *tbs_inst,
+					 const void *data, uint16_t length)
+{
+	const char *name = parse_string_value(data, length,
+					      CONFIG_BT_TBS_MAX_URI_LENGTH);
+
+	BT_DBG("%s", log_strdup(name));
+
+	if (tbs_client_cbs != NULL && tbs_client_cbs->friendly_name != NULL) {
+		tbs_client_cbs->friendly_name(conn, 0, tbs_inst->index, name);
+	}
+}
+
+/** @brief Handles notifications and indications from the server */
+static uint8_t notify_handler(struct bt_conn *conn,
+			      struct bt_gatt_subscribe_params *params,
+			      const void *data, uint16_t length)
+{
+	uint16_t handle = params->value_handle;
+	struct bt_tbs_instance *tbs_inst = lookup_inst_by_handle(conn,
+								       handle);
+
+	if (data == NULL) {
+		BT_DBG("[UNSUBSCRIBED] 0x%04X", params->value_handle);
+		params->value_handle = 0U;
+		if (tbs_inst != NULL) {
+			tbs_inst->subscribe_cnt--;
+		}
+
+		return BT_GATT_ITER_STOP;
+	}
+
+	if (tbs_inst != NULL) {
+		if (IS_ENABLED(CONFIG_BT_TBS_CLIENT_GTBS) && tbs_inst->gtbs) {
+			BT_DBG("GTBS");
+		} else {
+			BT_DBG("Index %u", tbs_inst->index);
+		}
+		BT_HEXDUMP_DBG(data, length, "notify handler value");
+
+		if (handle == tbs_inst->provider_name_handle) {
+			provider_name_notify_handler(conn, tbs_inst, data,
+						     length);
+		} else if (handle == tbs_inst->technology_handle) {
+			technology_notify_handler(conn, tbs_inst, data, length);
+		} else if (handle == tbs_inst->signal_strength_handle) {
+			signal_strength_notify_handler(conn, tbs_inst, data,
+						       length);
+		} else if (handle == tbs_inst->status_flags_handle) {
+			status_flags_notify_handler(conn, tbs_inst, data,
+						    length);
+		} else if (handle == tbs_inst->current_calls_handle) {
+			current_calls_notify_handler(conn, tbs_inst, data,
+						     length);
+		} else if (handle == tbs_inst->in_uri_handle) {
+			incoming_uri_notify_handler(conn, tbs_inst, data,
+						    length);
+		} else if (handle == tbs_inst->call_state_handle) {
+			call_state_notify_handler(conn, tbs_inst, data, length);
+		} else if (handle == tbs_inst->call_cp_handle) {
+			call_cp_notify_handler(conn, tbs_inst, data, length);
+		} else if (handle == tbs_inst->termination_reason_handle) {
+			termination_reason_notify_handler(conn, tbs_inst, data,
+							  length);
+		} else if (handle == tbs_inst->in_call_handle) {
+			in_call_notify_handler(conn, tbs_inst, data, length);
+		} else if (handle == tbs_inst->friendly_name_handle) {
+			friendly_name_notify_handler(conn, tbs_inst, data,
+						     length);
+		}
+	} else {
+		BT_DBG("Notification/Indication on unknown TBS inst");
+	}
+
+	return BT_GATT_ITER_CONTINUE;
+}
+
+static int tbs_client_common_call_control(struct bt_conn *conn, uint8_t inst_index,
+				   uint8_t call_index, uint8_t opcode)
```

Comment:
The split line alignment doesn't look right

Reply:
It wasn't :)